### PR TITLE
fix lazy-logging and wrong usage of raiseException

### PR DESCRIPTION
**Pull request by wdpypere**

### CODE REVIEW

File: lib/vsc/ldap/utils.py [hunk 1]
```diff
@@ -201,8 +202,9 @@ def search_sync(self, ldap_filter, base, attributes=None):
         try:
             res = self.ldap_connection.search_s(base, ldap.SCOPE_SUBTREE, ldap_filter, attributes)
         except ldap.LDAPError:
-            self.log.raiseException("Ldap sync search failed: base %s, ldap_filter %s, attributes %s",
+            self.log.exception("Ldap sync search failed: base %s, ldap_filter %s, attributes %s",
                                     base, ldap_filter, attributes)
```
Comment (stdweird) [inline lib/vsc/ldap/utils.py:206]: indentation

Comment (wdpypere) [reply lib/vsc/ldap/utils.py:206]: fixed

Comment (stdweird) [reply lib/vsc/ldap/utils.py:206]: is it?

File: lib/vsc/ldap/group.py [hunk 2]
```diff
@@ -76,8 +76,8 @@ def get_ldap_info(self):
         """Retrieve the data from the LDAP to initially fill up the ldap_info field."""
         group_ldap_info = self.ldap_query.group_filter_search("cn=%s" % (self.group_id))
         if len(group_ldap_info) == 0:
-            self.log.error("Could not find a group in the LDAP with the ID %s, raising NoSuchGroupError"
-                           % (self.group_id))
+            self.log.error("Could not find a group in the LDAP with the ID %s, raising NoSuchGroupError",
```
Comment (stdweird) [inline lib/vsc/ldap/group.py:79]: you can replace `self.log` with logging everywhere

Comment (wdpypere) [reply lib/vsc/ldap/group.py:79]: sure, but then the logger name is lost.

File: lib/vsc/ldap/utils.py (hunk 1 above)
Comment (wdpypere) [reply lib/vsc/ldap/utils.py]: I fixed another one, now it is fixed

File: lib/vsc/ldap/group.py (hunk 2 above)
Comment (wdpypere) [reply lib/vsc/ldap/group.py]: done

Comment (stdweird) [reply lib/vsc/ldap/group.py]: the logger name, well, it's not a big loss imho. we used to do that to make debugging easier, but tbh, i have rarely relied on the logname while debugging

**Pull request merged**